src/App.js

```diff
@@ -1,41 +1,17 @@
 import "./App.css";
 import logo from "./logo.png";
+import {Routes,Route,Link} from 'react-router-dom';
```

Comment:
Add whitespaces between comma, and braces.

Reply:
okay. Spacing is provided.